README.md

```diff
@@ -288,17 +345,19 @@ const useStoreA = create<StoreState>(
 Sometimes multiple state changes might happen in a short amount of time and you only want to store one change in history. To do so, we can utilize the `handleSet` callback to set a timeout to prevent new changes from being stored in history. This can be used with something like [`throttle-debounce`](https://github.com/niksy/throttle-debounce), [`just-throttle`](https://github.com/angus-c/just/tree/master/packages/function-throttle), [`just-debounce-it`](https://github.com/angus-c/just/tree/master/packages/function-debounce), [`lodash.throttle`](https://www.npmjs.com/package/lodash.throttle), or [`lodash.debounce`](https://www.npmjs.com/package/lodash.debounce). This a way to provide middleware to the temporal store's setter function.
 
 ```tsx
-const withTemporal = temporal<MyState>(
```

Comment:
It was initially confusing for me that these examples did not use `create` from `zustand`. I think it makes sense to keep a common code template for all ZundoOptions API examples, so this was updated here.